.github/workflows/ci.yml

```diff
@@ -4,14 +4,15 @@ on:
   pull_request:
   push:
   schedule:
-    - cron: '20 */2 * * *'
+    - cron: '20 2 * * *'
 
 jobs:
   unit:
     name: Templating
+    if: github.event_name != 'push' || github.ref == 'refs/heads/master' || github.ref == 'refs/heads/develop'
     runs-on: ubuntu-latest
     container:
-      image: atk4/image
+      image: ghcr.io/mvorisek/image-php
```

Comment:
could we keep atk4/image here?

Reply:
it will complicate merging, this is for build only